### ISSUE TITLE
i was working on a doc but now it won't load?

### ISSUE THREAD

**Issue by mint-a** [groupdocs-free-consulting/projects#20]
first of all i'm extremely computer-dumb and i don't know how groupdocs works. i just didn't wanna work on word on mobile so i looked up "docx editor" on google and groupdocs editor was the first thing that popped up. i just opened it in the internet browser that comes with your samsung mobile and i worked on a few different docx's that i created on my computer's word. now i don't know if i did something wrong- maybe the free browser option doesn't work for too long? but basically i didn't open those pages for a day or two and when i did open it the area where you write is just  blank. not _empty_, and it doesn't let me click on it or write anything, it's just _blank_. it says "please wait while document is loading" when you open or refresh the page, and after the file/format/insert tabs show up on top a little grey box pops up at the bottom that says "[object ProgressEvent] Unknown Error". i tried doing some research on google but nothing really turned up and i don't know what to do. some of those documents were quite important school and/or therapy things that i don't have saved anywhere else and i'll need them soon. if anyone can help and be real patient with my dumb arse, i'd literally owe you my life

**Comment by atirtahirgroupdocs** [comment]
@mint-a 

Please have a look at the following screenshot:

![screenshot](https://user-images.githubusercontent.com/15940608/141350417-72a912a1-6867-4153-a404-7e0028a55243.png)

We can successfully upload and edit Word document(s). Could you please share the problematic file(s)?

> some of those documents were quite important school and/or therapy things that i don't have saved anywhere else and i'll need them soon.

Could you please share more details on it? Did you upload and edit the document, but forgot to save it? If you didn't save it, all the editing is lost. 
Secondly, we'd recommend you to post GroupDocs app related issues on app [forum](https://forum.groupdocs.app/).

**Comment by mint-a** [comment]
this is how it shows up for me. there's none of the text that there used to be and i can't write/type where you have text, in fact that area's not even click-able
![Screenshot (463)](https://user-images.githubusercontent.com/94123089/141770385-6205f852-03a1-4de6-ad10-5fc19441b545.png)
unfortunately every docx's i was editing had personal info (full name, phone number etc) so if i can help it i'd rather not share them, the only thing without information like that is this [https://products.groupdocs.app/editor/it-it/wysiwyg/WordProcessing/49b14a61-7c1d-4167-a898-b583ace6ef1b/MENTA.docx](url) but the content is still intimate to say the least
i uploaded and edited documents, and i did save them (and by that i mean clicking on the save document icon under "file") after almost every edit that i made, but i didn't download any of the edited documents

sorry for posting in the wrong place, i thought i followed the instructions under support -> free consulting on the website. should i copy everything i posted so far on that link?
either way, thank you for your time !

**Comment by atirtahirgroupdocs** [comment]
@mint-a 

Please share MENTA.docx here and we'll further look into this scenario. 

> i uploaded and edited documents, and i did save them (and by that i mean clicking on the save document icon under "file") after almost every edit that i made, but i didn't download any of the edited documents
Please note that all such documents (that users upload) are removed from the server after 24 hours automatically.

**Comment by mint-a** [comment]
i'm sorry but what do you mean by sharing that? i only have that link and the word document that i originally uploaded to groupdocs editor

**Comment by atirtahirgroupdocs** [comment]
@mint-a 

Let me try to explain again, if you uploaded a source file and did some editing, saved it but forgot to download it, you cannot get it now. Because, the app server doesn't save file for a long time. Have a look at the following screenshot/error message. 
It clearly says that file doesn't exist.

![screenshot](https://user-images.githubusercontent.com/15940608/141816933-728cb81c-ff1a-4fdb-9931-d36b4b845788.png)


>  so if i didn't edit those documents for more than 24hrs they just got removed and there's no way to get them back?

It doesn't matter if you edit them or not. As it is a free web app, it doesn't store user's data. So the uploaded file is removed on it's own (edited or not). 

The best approach is to:

- Upload the document 
- Edit it
- Save the editing 
- Download updated/edited file

**Comment by mint-a** [comment]
ok well if that's the case that stuff is just lost and i don't want to waste your time further, thanks for the help though :)

**Comment by atirtahirgroupdocs** [comment]
@mint-a 

You are welcome.
